web/src/OverviewTable.stories.tsx

```diff
@@ -44,6 +47,32 @@ export default {
 
 export const TwoResources = () => <OverviewTable view={twoResourceView()} />
 
+export const TiltfileWarning = () => {
+  let view = nResourceView(1)
```

Comment:
(nitpick / optional) i found myself curious about how a resource with warnings would look with other resources around it.  
```suggestion
  let view = nResourceView(10)
```

Reply:
done